### PR TITLE
Provide better examples include/excludeassets, call out the defaults

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2235

### CODE REVIEW

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 1]
```diff
@@ -115,18 +115,23 @@ Allowable values for these tags are as follows, with multiple values separated b
...
-
 ```xml
 <ItemGroup>
     <!-- ... -->
-
+    <!-- Everything except the content files will be consumed by the project -->
+    <!-- Everythign except content files and analyzers will flow to the parent project-->
     <PackageReference Include="Contoso.Utility.UsefulStuff" Version="3.6.0">
-        <IncludeAssets>all</IncludeAssets>
+        <IncludeAssets>all</IncludeAssets> <!-- Default is `all`, can be omitted-->
```
Comment (JonDouglas) [inline docs/consume-packages/Package-References-in-Project-Files.md:124]: Should we move comment above for consistency?

**Review by JonDouglas — APPROVED** [review]
Thank you for the contribution! Comment to consider.

File: docs/consume-packages/Package-References-in-Project-Files.md (hunk 1 above)
Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md:124]: The way I saw it was that the 2 comments above talk about the overall behavior of the 3 props when combined, so the effective behavior. 
The inline one is referring specifically to `IncludeAssets`.

Comment (JonDouglas) [reply docs/consume-packages/Package-References-in-Project-Files.md:124]: cool, merge!

**Pull request merged**